### PR TITLE
Validate min-size and fix misaligned trade page layout

### PR DESCRIPTION
**Pull request by saml33**
Show an error onBlur of input when a user enters a trade size below the market min. Also noticed the default layout was not aligned on a couple of breakpoints so a fix for that is with this too.

### CODE REVIEW

**Comment by mschneider** [comment]
image

when the mouse is not on the graph itself but on the highlighted area the total value shows NaN.
when the mouse is outside the graph the total value is blurred.

can we instead show the current value in each case? it's all i want to see when I load the page at the beginning of the day :)

File: components/TradeForm.tsx [hunk 1]
```diff
@@ -227,19 +256,31 @@ export default function TradeForm() {
     }
   }
 
+  const validateInput = () => {
+    if (baseSize < market.minOrderSize) {
```
Comment (mschneider) [inline components/TradeForm.tsx]: market can be null

File: components/TradeForm.tsx [hunk 2]
```diff
@@ -227,19 +256,31 @@ export default function TradeForm() {
...
+        `Size must be greater than or equal to ${market.minOrderSize} ${baseCurrency}`
+      )
+    }
+  }
+
   const disabledTradeButton =
-    (!price && tradeType === 'Limit') || !baseSize || !connected || submitting
+    (!price && tradeType === 'Limit') ||
+    !baseSize ||
+    baseSize < market.minOrderSize ||
```
Comment (mschneider) [inline components/TradeForm.tsx]: market can be null

**Comment by saml33** [comment]
@mschneider – Added a check for market

**Pull request merged**